frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js

```diff
@@ -126,13 +126,11 @@ function getXAxisProps(props, datas, warn) {
 ///------------------------------------------------------------ DIMENSIONS & GROUPS ------------------------------------------------------------///
 
 function getDimensionsAndGroupsForScatterChart(datas) {
-  const dataset = crossfilter();
-  datas.map(data => dataset.add(data));
-
+  const dataset = crossfilter(datas);
```

Comment:
This is unrelated but it [did not seem to make sense not to pass](https://github.com/crossfilter/crossfilter/wiki/API-Reference) `datas` while creating crossfilter dataset